src/shared/helpers/case-type-sidebar-tabs-flags.js

```diff
@@ -13,6 +13,9 @@ const caseTypeSidebarTabsFlags = {
     FOI: {
         people: false,
         foi_actions: true
+    },
+    BF: {
+        foi_actions: true
```

Comment:
I know this is unrelated and a side effect of how actions was genericised. Could you raise a ticket to generalise this please...

Reply:
I've raised a ticket to change this to just `actions`